bde_fdw_rds/bde_fdw_rds_stack.py

```diff
@@ -0,0 +1,28 @@
+from aws_cdk import Stack, aws_ec2
+from constructs import Construct
+
+
+class BdeFdwRdsStack(Stack):
+    def __init__(  # type: ignore[no-untyped-def]
+        self,
+        scope: Construct,
+        construct_id: str,
+        vpc_id: str,  # vpc id
+        subnet_ids: list[str],  # list of subnet ids
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        # ----- Networking -----
+        aws_ec2.Vpc.from_lookup(self, "ExistingVPC", vpc_id=vpc_id)
```

Comment:
I guess the actual lookup isn't going to be "ExistingVPC". Is this information missing?

Reply:
Have updated to `BDEHostVPC`